src/database/firebase.tsx

```diff
@@ -18,6 +19,120 @@ const config = {
   appId: APP_ID,
 };
 
-firebase.initializeApp(config);
+!firebase.apps.length ? firebase.initializeApp(config) : firebase.app();
 
+const database = firebase.firestore();
```

Comment:
this should be handled by #15, I would pull from main

Reply:
A lot of this is because we originally branched off of #15  before it was merged. Git just didn't update the diffs correctly